reference/kubernetes.md

```diff
@@ -111,12 +88,12 @@ $ kubectl config \
     --user=my-user
 ```
 
-If you have done this and are using the default context file, you will be able to set the configuration variable
-`kubernetes:configContext` to the given context name:
+If you have done this and are using the default context file, you will be able to set the
+configuration variable `kubernetes:config:context` to the given context name:
```

Comment:
Is `:config:` really needed?  We've gotten rid of the need for that in all other providers.